docs/_includes/body-blog.html

```diff
@@ -0,0 +1,60 @@
+<div class="page-wrapper" tabindex="-1" role="main">
```

Comment:
I copied things around. 

Code is not the cleanest, but we already have a task to cleanup UI, remove unused templates etc — we can take care of that when working on that part.

This is still an incremental improvement in the right direction.